src/sycl/sycl_stream_submit_cpu_primitive.cpp

```diff
@@ -64,21 +64,16 @@ using make_kernel_tag
 template <typename... param_types>
 status_t submit_cpu_primitive_with_params_impl(submit_ctx_t *submit_ctx,
         cl::sycl::handler &cgh, param_types... params) {
-    // Trick the compiler by capturing scalar values in the kernel
-    // instead of pointers what is not allowed.
-    uintptr_t submit_ctx_ptr = reinterpret_cast<uintptr_t>(submit_ctx);
-    using tag_type = make_kernel_tag<param_types...>;
```

Comment:
Since this is no longer used the related code should also be removed:

- https://github.com/oneapi-src/oneDNN/blob/3cee57914179b3929ce9153fc5c6376407cc0896/src/sycl/sycl_stream_submit_cpu_primitive.cpp#L37-L39
- https://github.com/oneapi-src/oneDNN/blob/3cee57914179b3929ce9153fc5c6376407cc0896/src/sycl/sycl_stream_submit_cpu_primitive.cpp#L60-L62